packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -43,11 +43,26 @@ describe('Rating', () => {
                 });
 
                 // Act & Assert
-                expect(wrapper.vm.getRatingStarPercentage).toBeDefined();
+                expect(wrapper.vm.getRatingDescription).toBeDefined();
             });
 
             describe('when invoked', () => {
-                it('should return a correct calculated number from a combination of `starRating` and `maxStarRating`', () => {
+                it('should return a singular description if the rating is less than 2', () => {
+                    // Arrange
+                    propsData = {
+                        starRating: 1
+                    };
+                    wrapper = shallowMount(VRating, {
+                        propsData,
+                        localVue,
+                        i18n
+                    });
+
+                    // Act & Assert
+                    expect(wrapper.vm.getRatingDescription).toMatchSnapshot();
```

Comment:
I know this test wasn't actually changed as part of this PR, but IMO a description is fairly simple so the expected text should live within the test so it's easy to see at a glance. For large and complex objects, snapshots are great.

Also, how does this work for decimals? I would expect `1.8 stars` rather than `1.8 star`. Might be worth raising a ticket to look into this.

Reply:
Originally had the text instead of snapshot. Will change back. With the decimals that's a good spot. Ill add a ticket for this.